src/org/safs/rest/service/models/consumers/SafsrestAdapter.groovy

```diff
@@ -50,22 +54,17 @@ import org.safs.rest.service.models.providers.SafsRestPropertyProvider
 class SafsrestAdapter extends ClientPOJOAdapter {
     public static final USERID = "userid"
     public static final PASSWORD = "password"
-
-	/** an Auth object holding the necessary information to authenticate and get authorization. */
-	public Auth auth
```

Comment:
If we keep this field, could we remove fields
    public trustedUserid
    public trustedPassword
    public tokenProviderServiceName
    public tokenProviderAuthTokenResource

which will make the code in Service simpler. Not sure if it is better or worse.